apps/frontend/src/pages/contributors/libs/components/contributor-merge-form/contributor-merge-form.tsx

```diff
@@ -75,8 +75,8 @@ const ContributorMergeForm = ({
 			<div className={styles["warn-message"]}>
 				<Icon height={16} name="warning" width={16} />
 				<span className={styles["warn-message-text"]}>
-					The selected contributor will be deleted. This action cannot be
-					undone.
+					The selected contributor will be deleted. This action can be undone
+					later using the split functionality.
```

Comment:
I think it sounds a little bit unclear, perhaps something like this is better? 

> The selected contributor will be deleted. You can undo this action later using the split functionality.

Reply:
this message was attached to my ticket, I took it from the task